target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4018-ea6350v3.dts

```diff
@@ -0,0 +1,287 @@
+/* Copyright (c) 2015, The Linux Foundation. All rights reserved.
```

Comment:
Please check out https://openwrt.org/submitting-patches#dts_checklist and add a SPDX-License-Identifier of your choosing.